frontend/src/pages/documents/components/document.create-dialog.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <create-dialog :opened="opened" :emptyTitle="emptyTitle">
+      <template v-slot:title>
+        <label for="document-title">Enter new document title</label>
+        <input type="text" id="document-title" v-model="title" autocomplete="off" />
+      </template>
+      <link-button @click="createdDocument">Create</link-button>
+      <link-button @click="toggleCreateDialog('')" v-on:click="hideMessage">Close</link-button>
+    </create-dialog>
+  </div>
+</template>
+
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+import { authVuexModule, documentsVuexModule } from '@src/vuex/store-accessor';
+import CreateDialog from '@components/dialogs/create-dialog.vue';
+import LinkButton from '@components/buttons/link-button.vue';
+import dateAndTime from '@src/date-time/dateAndTime';
+
+export default defineComponent({
+  props: {
+    opened: {
+      type: Boolean,
+      required: true,
+    },
+    id: {
+      type: String,
+      required: true,
+    },
+    empty: {
+      type: Boolean,
+      required: true,
+    },
+  },
+
+  components: {
+    CreateDialog,
+    LinkButton,
+  },
+
+  data() {
+    return {
+      title: '',
+      emptyTitle: this.empty,
+      toggle: this.toggleCreateDialog,
+    };
+  },
+
+  methods: {
+    createdDocument(): void | undefined {
+      if (this.title === '') {
+        this.emptyTitle = true;
+        return;
+      }
+      this.emptyTitle = false;
+      this.toggle();
+      const id = this.createID();
+      const currentUser = authVuexModule.currentUser;
+      if (currentUser) {
```

Comment:
There is no need need to always check if is currentUser exists

Reply:
Added the non-null assertion operator to get rid of this check